indexer/api/api_test.go

```diff
@@ -0,0 +1,93 @@
+package api_test
```

Comment:
typically the test package will be the same. That ways you don't have to export all the fields on `Api`. They can remain private but visible to the tests

Reply:
Oh nice that's way better!